proto/xudrpc.proto

```diff
@@ -197,6 +197,8 @@ message Order {
   int64 created_at = 8 [json_name = "created_at"];
   // Lightning invoice
   string invoice = 9 [json_name = "invoice"];
+  // Indicates whether an order was cancelled 
+  bool cancelled = 10 [json_name = "cancelled"];
```

Comment:
The American in me thought this was spelled "canceled" and upon googling it turns out that both are correct and "cancelled" is the British spelling while "canceled" is the American spelling. I think we should change this though because we're already using "canceled" elsewhere in the code (see `CancelOrderResponse`) but not "cancelled" and we should stick to one spelling convention. "cancelled" is also used a few other places in this PR so just do a find all.